lib/octokit/client/reviews.rb

```diff
@@ -0,0 +1,174 @@
+module Octokit
+  class Client
+
+    # Methods for the Reviews API
+    #
+    # @see https://developer.github.com/v3/reviews/
+    module Reviews
+
+      # List reviews on a pull request
+      #
+      # @param repo [Integer, String, Hash, Repository] A GitHub repository
+      # @param id [Integer] The id of the pull request
+      # @see https://developer.github.com/v3/pulls/reviews/#list-reviews-on-a-pull-request
+      #
+      # @example
+      #   @client.pull_request_reviews('octokit/octokit.rb', 2)
+      #
+      # @return [Array<Sawyer::Resource>] Array of Hashes representing the reviews
+      def pull_request_reviews(repo, id, options = {})
+        options = ensure_api_media_type(:reviews, options)
+        get "#{Repository.path repo}/pulls/#{id}/reviews", options
+      end
+
+      # Get a single review
+      #
+      # @param repo [Integer, String, Hash, Repository] A GitHub repository
+      # @param pull_id [Integer] The id of the pull request
```

Comment:
Can we update all of the uses of `pull_id` to use `number` instead. It makes it consistent with the rest of the API methods.